### ISSUE TITLE
[Accessibility] Lack of clarity when navigating

### ISSUE THREAD

**Issue by lpalmaro** [Polymer/polymer-element-catalog#109]
it's not clear with a screen reader that you can click on the tiles (Md, Fe, etc.). They should potentially be implemented as links or buttons. +aboxhall for any additional suggestions. 

Tested: 
"Chrome OS / ChromeVox
Mac / VoiceOver
Win / NVDA"

**Comment by azakus** [comment]
#105 should be partially solved with this

**Comment by mbleigh** [comment]
This was likely solved or at least helped by bcc27886
